src/components/info/StationMetadata/StationMetadata.js

```diff
@@ -3,102 +3,170 @@
 // passed into React Table.
 
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React from 'react';
 import ReactTable from 'react-table';
-import find from 'lodash/fp/find';
+
+import flow from 'lodash/fp/flow';
+import map from 'lodash/fp/map';
+
 import FrequencySelector from '../../selectors/FrequencySelector';
 import logger from '../../../logger';
+import {
+  stationNetwork,
+  uniqStationFreqs,
+  uniqStationLocations,
+  uniqStationNames,
+  uniqStationObsPeriods,
+  uniqStationVariableNames
+} from '../../../utils/station-info';
 
 import 'react-table/react-table.css';
 import './StationMetadata.css';
 
 
 logger.configure({ active: true });
 
+// TODO: Put these in a util module
 const formatDate = d => d ? d.toISOString().substr(0,10) : 'unknown';
 
+const lexCompare = (a, b) => {
```

Comment:
This is for sorting in the metadata table?

Reply:
Yes, it is for sorting lists of things, e.g., lists of names, obs freqs. These arise because a station can have many histories, hence many names, obs freqs, etc. (These lists are reduced to their unique elements.) This implements lexicographic ordering for lists.

I plan to document all this in the code / README before I merge.